examples/gallery/simple_differential_rotation.py

```diff
@@ -0,0 +1,72 @@
+"""
+============================
+Simple Differential Rotation
+============================
+
+The Sun is known to rotate differentially, meaning that the rotation rate
+near the poles (rotation period of approximately 35 days) is not the same as
+the rotation rate near the equator (rotation period of approximately 25 days).
+This is possible because the Sun is not a solid body. Though it is still poorly
+understood, it is fairly well measured and must be taken into account
+when comparing observations of features on the Sun over time.
+A good review can be found in Beck 1999 Solar Physics 191, 47–70.
+This example explores the effects of differential rotation.
```

Comment:
My $0.02 - include the qualifier "solar" in front of every time "differential rotation" is mentioned to make explicit that solar differential rotation is being considered, and not some other arbitrary differential rotation.